Archer/Program.cs

```diff
@@ -1,9 +1,74 @@
-﻿namespace Archer;
+﻿using System.Runtime.Intrinsics.Arm;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Archer;
 
 class Program
 {
     static void Main(string[] args)
     {
-        Console.WriteLine("Hello, World!");
+        try
+        {
+            StartSimulation();
+        }
+        catch (Exception exception)
+        {
+            Console.WriteLine(exception.Message);
+        }
+    }
+
+    private static void StartSimulation()
+    {
+        Archer robin = new Archer("Robin", 10);
+
+        while (true)
+        {
+            Console.WriteLine("\n--- Archer Menu ---");
+            robin.DisplayStatus();
+            Console.WriteLine("1 - Shoot");
+            Console.WriteLine("2 - Add arrows");
+            Console.WriteLine("3 - Quit");
+            Console.WriteLine("Please enter a number (1-3)");
+
+            string input = Console.ReadLine();
+
+            switch (input)
+            {
+                case "1":
+                    robin.Shoot();
+                    break;
+
+                case "2":
+                    robin.AddArrows(LoadIntegerFromConsole("Enter the number of arrows:"));
+                    break;
+
+                case "3":
+                    return;
+
+                default:
+                    Console.WriteLine("Invalid input. Please try again.");
+                    continue;
+            }
+        }
+    }
+
+    public static int LoadIntegerFromConsole(string alert)
```

Comment:
Super pomocna metoda 👍 Jen bych nejak zdokumentoval, ze nas necha pouze nacist kladna cisla:

```suggestion
    public static int LoadPositiveIntegerFromConsole(string alert)
```